docs/docs-content/clusters/edge/site-deployment/site-installation/cluster-deployment.md

```diff
@@ -55,11 +60,20 @@ Use the following steps to create a new host cluster so that you can add Edge ho
 
 9. The node configuration page is where you can specify what Edge hosts make up the host cluster. Assign Edge hosts to the **master-pool** and the **worker-pool**. When you have completed configuring the node pools, click on **Next**.
 
+10. (Optional) When you assign Edge hosts to node pools, you can optionally specify a static IP address for each Edge host. If you want to specify a static IP, toggle on **Static IP** and provide the following information:
+    
+    -  IP address for the Edge host
+    -  IP address for the default gateway
+    -  Subnet mask of your network
+    -  IP address of your DNS server
```

Comment:
We usually use a table with the columns "Field", and "Description" to explain fields. Use the description to provide more context.